packages/docs/docs/02-Guides/03-Transactions/04-Replaced Transactions.mdx

```diff
@@ -0,0 +1,10 @@
+# Replaced Transactions
+
+If the status is pending you can replace transaction with increased gas limit to have it mined quicker. To do it in just send another transactions with the same nonce as previous. This transaction is called `replaced transaction`.
+After replacing transaction on wallet, you will see it as a previous transaction(one you replaced) and it will be treated as a previous transaction.
+
+<img src="https://lh3.googleusercontent.com/FtUnvBTcWuTDZauj0IXI4uY9kB3gxkVHJFsuu5SlTzs0zRt8RNV3GAup9WGrKfw1AoALuoFYM5ZjT320wR2mS3UOozJl8bhisQ2d5Bh0aDqe58x9rzvEYbZbG1OtJm7_HdyBBXs7" alt="replace_metamask"/>
```

Comment:
Hotlinking is bad - copy the image and put it in our repo.

Reply:
Okay